app/views/landing.html.erb

```diff
@@ -0,0 +1,49 @@
+<% content_for :content do %>
+
+<div class="container mx-auto flex flex-col justify-center items-center px-6 pt-12 pb-20">
+
+	<nav class="m-2 flex justify-center">
+		<div class="logo">
+			<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 551 75" width="551" height="75" style="max-width: 220px;">
```

Comment:
ideally we can extract this to `public/logo.svg` and then consume here as `<img src="plublic/logo.svg">` (that URI might be wrong)